### DISCUSSION TITLE
Airflow helm chart to support separate labels for each Deployment

### DISCUSSION THREAD

**Discussion by mayankbatra123** [apache/airflow#23828] [General] [answered]
It has been came to notice that for airflow helm chart - 1.5.0 labels for each deployment (webserver , worker etc.) are combined and passed via values.yaml;  as - **".Values.labels".**

**In Helm Chart** - 
_{{- with .Values.labels }}
{{ toYaml . | indent 4 }}
{{- end }}_

This will span same labels across all deployments, which becomes a problem sometimes if separate labels are needed.

**Suggestion** -
**Values.yaml should support separate labels for each deployment.**
Such as :
_{{- with .Values.webserver.labels }}
{{ toYaml . | indent 4 }}
{{- end }}_

**Comment by potiuk** [comment, marked as answer]
Can you make a PR with a proposed change please?

**Reply by mayankbatra123** [reply]
https://github.com/apache/airflow/pull/24668
Sharing same on Slack as well !!
